### ISSUE TITLE
About the losses used for the pre-trained model.

### ISSUE THREAD

**Issue by LynnHo** [facebookresearch/FaderNetworks#5]
Does anybody know whether or not the pre-trained model is trained with the PatchGAN discriminator loss and the classifier loss?

**Comment by glample** [comment]
No, these discriminators have not been used. But properly tuned they might be helpful (especially the PatchGAN discriminator) so we implemented them and let them as optional for people interested.

**Comment by glample** [comment]
Yes, you can specify several attributes during training, by setting the `attr`attribute to `"Smiling,Male"`, for instance, or any list of attributes separated by a comma. However, in practice, adding many attributes makes the output more blurry. You will also need to modify a bit the `interpolate.py` script to swap multiple attributes, we did not include that in the repo.
